content/recipes/authoring/kata-snippets/preloaded.md

```diff
@@ -21,4 +21,4 @@ Failure to comply with the below guidelines should be considered an issue to be
 - Preloaded Code snippet is a code, and, as such, should conform to [Codewars General Coding Guidelines](/recipes/authoring/general/).
 - **Do not put in preloaded any code which should not be called by submitted solution.** Code placed in this snippet can be called (and for some languages even modified or redefined) by submitted solution, and any data stored here can be read or modified programatically. It should not contain anything critical what would be used by tests to validate correctness of the solution.
 - **Be aware that user solution is able to read the file with preloaded snippet**. It should not contain any information which would reveal details on solution or tests, which should not be visible to the user.
-- **Do not hide useful information in preloaded.** Preloaded snippet can be use to reduce boilerplate, and its code is available to all other parts of solution. It can be tempting to put there some data or types which should be shared among user solution, sample tests, and submission tests. But remember that contents of preloaded code are not displayed anywhere in user interface and users cannot see it. This is especially problematic for users who would like to work on the kata in their local IDE, but now are missing some pieces necessary to set up their local environment. If you use preloaded part to reduce bolerplate code, consider adding necessary information also to kata description, preferrably in [language conditional block](https://docs.codewars.com/references/markdown/extensions/).
\ No newline at end of file
+- **Do not hide useful information in preloaded.** Preloaded snippet can be use to reduce boilerplate, and its code is available to all other parts of solution. It can be tempting to put there some data or types which should be shared among user solution, sample tests, and submission tests. But remember that contents of preloaded code are not displayed anywhere in user interface and users cannot see it. This is especially problematic for users who would like to work on the kata in their local IDE, but now are missing some pieces necessary to set up their local environment. If you use preloaded part to reduce bolerplate code, consider adding necessary information also to kata description, preferrably in [language conditional block](/references/markdown/extensions/).
```

Comment:
```suggestion
- **Do not hide useful information in the "Preloaded Code" snippet.** The "Preloaded Code" snippet can be used to reduce boilerplate, and its code is available to all other parts of the solution. It can be tempting to put there some data or types which should be shared among the user solution, sample tests, and submission tests. But remember that the contents of the "Preloaded Code" snippet are not displayed anywhere in the user interface and users cannot see it. This is especially problematic for users who would like to work on the kata in their local IDE, but now are missing some pieces necessary to set up their local environment. If you use the "Preloaded Code" snippet to reduce boilerplate code, consider adding necessary information to the kata description as well, preferably in a [language conditional block](/references/markdown/extensions/).
```